CMakeLists.txt

```diff
@@ -128,6 +128,7 @@ rocm_enable_clang_tidy(
         -UNDEBUG
         -DMIGRAPHX_USE_CLANG_TIDY
         "-Dmain\\\\(...\\\\)=main\\\\(__VA_ARGS__\\\\) // NOLINT"
+        -Xclang -fallow-half-arguments-and-returns
```

Comment:
These flags should be added to the migraphx_device target not to clang-tidy.

Reply:
fixed